src/main/scala/nl.knaw.dans.easy.deposit/EasyDepositApiApp.scala

```diff
@@ -57,6 +58,13 @@ class EasyDepositApiApp(configuration: Configuration) extends DebugEnhancedLoggi
     logger.info(s"users.ldap-user-id-attr-name = $ldapUserIdAttrName")
     logger.info(s"users.ldap-admin-principal = $ldapAdminPrincipal")
   }
+  val multipartConfig: MultipartConfig = MultipartConfig(
+    location = Option(properties.getString("multipart.location", null)),
+    maxFileSize = Option(properties.getLong("multipart.max-file-size", null)),
+    maxRequestSize = Option(properties.getLong("multipart.max-request-size", null)),
+    fileSizeThreshold = Some(properties.getInt("multipart.file-size-threshold", 0)),
+    // getInt doesn't compile with null as default
```

Comment:
would it solve this remark if you did something like `Option(properties.getInt("multipart.file-size-threshold")).getOrElse(0)`? Not sure, haven't tried it.